### PR TITLE
Updating nstx-fidasim repo

### PR DESCRIPTION
**Pull request by alvin-garcia**

### CODE REVIEW

File: src/nstx_prefida.pro [hunk 1]
```diff
@@ -9,8 +9,9 @@ PRO nstx_prefida, inputs, igrid=igrid,bgrid=bgrid
                    n_halo:500000L,n_dcx:500000L,n_birth:10000L,$
                    ne_wght:50,np_wght:50,nphi_wght:100,emax_wght:100.0d0,$
                    nlambda_wght:1000,lambdamin_wght:647.d0,lambdamax_wght:667.d0,$
-                   calc_npa:0,calc_brems:1,calc_bes:1,calc_fida:1,$
-                   calc_birth:0,calc_fida_wght:0,calc_npa_wght:0,dump_dcx:0,$
+                   calc_npa:2,calc_brems:1,calc_fida:1,calc_neutron:1,$
+                   calc_nbi:1,calc_dcx:1,calc_halo:1,calc_cold:1,$
+                   calc_birth:1,calc_fida_wght:1,calc_npa_wght:1,calc_pfida:1,calc_pnpa:2,$
```
Comment (lstagner) [inline src/nstx_prefida.pro]: see comments on d3d-fidasim

**Comment by alvin-garcia** [comment]
Don't merge, I still need to incorporate NPA geometry to nstx_prefida

**Comment by lstagner** [comment]
Yeah, looking at the write_npa_file it looks like I just stopped midway.

**Comment by alvin-garcia** [comment]
Okay, please review now.

**Comment by lstagner** [comment]
Since the write_npa_file.pro doesn't actually write the npa geometry to an HDF5 file (Like the other write_* files) I would just rename it to `npa_chords` or something like that. Also, `nstx_prefida` still doesn't support the NPA diagnostics

**Comment by lstagner** [comment]
rename the `write_npa_chords` function to `nstx_npa`

**Comment by alvin-garcia** [comment]
Anything else?

**Comment by lstagner** [comment]
add a NPA diagnostic to the `nstx_inputs.pro` function. Make sure its one that views the same beam as the `active_vfida`

**Comment by alvin-garcia** [comment]
Luke, I have updated the nstx repo. Please review, thanks.

**Comment by lstagner** [comment]
This is fine

**Pull request merged**